### DISCUSSION TITLE
Different Arming Modes Available

### DISCUSSION THREAD

**Discussion by ez4nick** [node-alarm-dot-com/homebridge-node-alarm-dot-com#113] [General]
First want to say thanks for this awesome integration, I am new to this world and amazed with what is possible. 

I have everything setup and it works with my security system configuration, but am wondering about the different arming modes available. I see "Home", "Away", "Night" and "Off". Is there any way to customize the list as in only show certain ones, or even have the ability to rename the listed options? I am used to seeing "Away", "Stay" or "Disarm" as the choices in the app I normally use and on the panel itself. It seems like "Home" and "Night" are almost the same thing, "Away" makes sense, and "Off" is Disarm. Thanks!

**Comment by chase9** [comment]
Glad the plugin is working well for you! As far as I know, the alarm states are hard-coded into homekit and can't be changed. Some people take advantage of the separation by making different modes do different things. For example, you could make night mode bypass or silent arm but not home mode. This can be customized in the plugin settings.

Hope this helps!
